Refresh.GameServer/Extensions/MemoryStreamExtensions.cs

```diff
@@ -0,0 +1,8 @@
+using System.Text;
+
+namespace Refresh.GameServer.Extensions;
+
+internal static class MemoryStreamExtensions
+{
+    internal static void WriteString(this MemoryStream ms, string str) => ms.Write(Encoding.Default.GetBytes(str));
```

Comment:
`Encoding.Default` can vary from computer to computer, and [even from the same computer at different times](https://learn.microsoft.com/en-us/dotnet/api/system.text.encoding.default?view=net-8.0) 
I would reccomend manually specifying `UTF8` (since thats *probably* what you mean anyway)

Reply:
Really? I've had issues with UTF8 encoding dropping binary characters as 0xEFBDBF...

I'm not exactly sure how this is possible given that the same docs you linked mention returning `UTF8Encoding`.

I'll test this out in a bit

Reply:
Worked just fine, whatever issue I was having years ago was probably solved